proto/profile/tx.proto

```diff
@@ -21,3 +23,12 @@ message MsgCreateCoordinator {
 message MsgCreateCoordinatorResponse {
   uint64 coordinatorId = 1;
 }
+
+message MsgUpdateCoordinatorDescription {
+  string address = 1;
+  CoordinatorDescription description = 2;
+}
+
+message MsgUpdateCoordinatorDescriptionResponse {
+  uint64 coordinatorId = 1;
```

Comment:
```suggestion
```

We can have an empty response since we already know the coordinator ID at this point or it can be queried with `coordinator-from-address`

The message response should contain data that are generated from the message logic